src/current/v23.2/cutover-replication.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Cut Over from a Primary Cluster to a Standby Cluster
```

Comment:
@stevendanna What do you think about the usage of "cut over" throughout these docs? Before Liv left, we discussed that cut over was preferable to "fail over", and "cut over" was generally a known in this context. Do you think it presumes traffic management too, and we should avoid it overall?

Reply:
I was thinking about that when reading the doc and don't have a great answer.  Personally, I like the word "promote." You "promote a standby to a primary."  But "cut over" has the advantage of it also being the words we used in some error messages, UI elements, and log messages.

As long as we make it clear in the doc that the user is responsible for ensuring traffic goes to the new cluster and that writes that go to the old cluster may be lost (which I think you do mention below). Then perhaps we don't have to get it perfect in this version.

In 24.1 when we have a way to go back quickly, I think we'll be forced to decide on terminology since then we'll need to decide if it is "failover and failback" or "cutover and cutback.

Which is a long way to say, perhaps stick with your current wording for now.

Reply:
OK, I understand where you're coming from. I have left as "Cutover / to cut over" for now. I honestly prefer "promote" from a language pov. However, if v24.1 is going to include a way to return quickly, and we'll be picking from fail/cut anyway... maybe this should stay.

I have made the note more of a warning re the user having to switch traffic manually.